zabbix/bin/zabbix-plugin.py

```diff
@@ -32,12 +42,12 @@
 
 # Generic call to the Rudder API, completed by its path param. Using URL and token obtained from zabbix.conf
 def getRequestToRudderAPI(conf, path):
-    try:
-        data = requests.get(conf["RUDDER"]["rudderAPIURL"] + path, headers={ 'X-API-Token': conf['RUDDER']['rudderAPIToken'] }, verify=False).json()
-    except:
-        print("[!] Error : check your Rudder API token")
-        sys.exit(-1)
-    return data
+        try:
```

Comment:
We should use the same indentation (4 spaces) everywhere

Reply:
Indentation is still not fixed, you have 8 spaces while the rest of the program has 4.